contracts/MarketOracle.sol

```diff
@@ -48,6 +50,13 @@ contract MedianOracle is Ownable, IOracle {
         reportExpirationTimeSec = reportExpirationTimeSec_;
     }
 
+    function setReportSecurityDelaySec(uint256 reportSecurityDelaySec_)
```

Comment:
Personally in favor of not making this parameter configurable. One less parameter to worry about.

Reply:
I was also considering having a higher value for PCE oracle.

Reply:
I'm not sure this value is going to change over time. If we want different security delays for different types of oracles we should pass it as a parameter to the constructor.

Reply:
Good idea, Same for expiration timestamp will be much longer for CPI

Reply:
Created a constructor.